code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -282,6 +294,7 @@
 /obj/item/gun_upgrade/scope/watchman
 	name = "Frozen Star \"Watchman\" scope"
 	desc = "Scope that can be attachet to avarage gun."
+	matter = list(MATERIAL_PLASMAGLASS = 2, MATERIAL_PLASTEEL = 2)
```

Comment:
a little fancy for a extremely basic scope, could just be plastic and glass. its a low zoom scope not a targeting system